README.md

```diff
@@ -30,16 +29,25 @@ It provides cross-service call chain tracking and performance information collec
 ### Architecture Diagram
 ![image](./doc/images/EaseAgent-Architecture.jpg)
 
-## Features
+#### Description
+Use `Byte Buddy` to modify the bytecode, insert the code that needs to be run before and after the method is executed, refer to the following code usage: 
```

Comment:
Leverage `Byte buddy` to modify bytecodes of classes dynamically, inject codes that.....

what do you think?